lib/activerecord-mysql-structure/active_record/structure_sql_sanitizer.rb

```diff
@@ -86,5 +122,36 @@ def sort_columns(lines)
 
       sorted_lines
     end
+
+    def sort_indices(lines)
+      indices = []
+      sorted_lines = []
+
+      lines.each do |line|
+        if UNSUPPORTED_ENGINE_REGEX.match?(line)
+          raise 'MySQL Structure Sanitizer detected an unsupported engine; aborting'
+        end
+
+        match = INDEX_REGEX.match(line)
+        if match
+          # Ensure trailing commas on each line
+          sanitized_index = "#{match[1]},#{match[3]}"
```

Comment:
Might want to used named captures: https://www.leighhalliday.com/named-captures-ruby-regular-expressions

Reply:
I like it!